pyproject.toml

```diff
@@ -54,6 +54,9 @@ mkdocs-git-revision-date-plugin = "^0.3.1"
 mike = "^0.6.0"
 mypy = "^0.931"
 mkdocs-material = "^8.2.7"
+mypy-boto3-secretsmanager = "^1.21.34"
+mypy-boto3-ssm = "^1.21.34"
+mypy-boto3-appconfig = "^1.21.34"
```

Comment:
@ran-isenberg @heitorlessa - i do wonder how we can run the unit tests without `mypy-boto3-*` being installed. Then we can be certain the missing imports don't cause runtime errors. Maybe we should make a note of this?

later integration test would help solve this gap.